examples/package.json

```diff
@@ -28,8 +28,7 @@
     "parcel": "1.12.4",
     "parcel-plugin-less-js-enabled": "1.0.2",
     "prettier": "2.1.1",
-    "react-code-prettify": "0.0.1",
     "tslint": "6.1.3",
-    "typescript": "4.0.2"
+    "typescript": "3.8.3"
```

Comment:
Is there a problem with TS 4?

Reply:
No problem as such, but the latest Meteor version uses TS 3.8.3.
It's easier to keep things in sync if we use the same version everywhere.

(I really wouldn't like to run into situations where our code is working fine in half of the situations, but produces inexplicable errors in the other half of the situations...)